src/Utility/HttpResponsePaginator.php

```diff
@@ -236,9 +238,11 @@ private function getNextResults(): bool
         if ($lastBuilder !== null && $this->lastResponse() !== null) {
             // Ensure basic pagination details are included in the request.
             if ($this->usingCheckpointPagination) {
+                // @codeCoverageIgnoreStart
```

Comment:
Just curious why this isn't testable / why ignore coverage for this block?

Reply:
There's no circumstance under which I'm able to trigger it; by all accounts it should never fire. I'm going to give this class a once over before RC/GA to see if it's even necessary, but I'm leaving it in there for the moment out of an abundance of caution.